fuzzers/065-gtp-common-pips/generate.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+from prjxray.segmaker import Segmaker
+import os
+import os.path
+
+
+def bitfilter(frame, word):
+    word = int(word / 32)
+
+    if frame not in [0, 1, 2, 3, 4, 5, 6, 7]:
```

Comment:
Please add a comment on what is going on with these?

Reply:
Done